packages/gafl-webapp-service/src/pages/concessions/benefit-ni-number/benefit-ni-number.njk

```diff
@@ -6,21 +6,20 @@
     set errorMap = {
         'ni-number': {
             'string.empty': { ref: '#ni-number', text: 'You have not told us your national insurance number' },
-            'string.max': { ref: '#ni-number', text: 'You national insurance number is longer than 15 characters' }
+            'string.pattern.base': { ref: '#ni-number', text: 'This does not look like a valid UK national insurance number, please check you have entered it correctly' }
         }
     }
 %}
 
 {% block pageContent %}
     <p class="govuk-body-m">We use this information to find out if you are eligible for a disabled concession</p>
-
     {{ govukInput({
       label: {
         text: "National Insurance number"
       },
       id: "ni-number",
       name: "ni-number",
-      hint: { html: "<p>You can find this on your payslip, P60 or tax return</p><p>For example, ‘QQ 12 34 56 C’</p>" },
+      hint: { html: "<p>You can find this on your payslip, P60 or tax return</p><p>For example, ‘NH 12 34 56 C’</p>" },
```

Comment:
GDS guidance states to use QQ 12 34 56 C as the example because it is not valid.

Reply:
done